runbot_travis2docker/models/runbot_build.py

```diff
@@ -79,11 +79,18 @@ def _job_20_test_all(self, build, lock_path, log_path):
             build._get_docker_run_cmd(), lock_path, log_path, cpu_limit=1200,
         )
 
-    def _job_21_coverage(self, build, lock_path, log_path):
+    def _job_21_coverage_html(self, build, lock_path, log_path):
         if not build.branch_id.repo_id.is_travis2docker_build:
             return super(RunbotBuild, self)._job_21_coverage(
```

Comment:
@moylop260 I merged prematurely: https://travis-ci.org/OCA/runbot-addons/jobs/387868156#L2714 - could you change this here too?

Reply:
oops sorry... https://github.com/OCA/runbot-addons/pull/161